### PR TITLE
Consolidate content on Azure metrics pages

### PR DESCRIPTION
**Pull request by alaudazzi**
## Proposed commit message

This PR:

- Removes duplicate and confusing content related to Kibana configurations, authentication, and costs. 
- Adds references to generic instructions documented on the [Azure Resource Metrics](https://docs.elastic.co/integrations/azure_metrics#setup) page.
- Adds the `Requirements` and `Setup` sections to comply with the [documentation guidelines](https://github.com/elastic/integrations/blob/main/docs/documentation_guidelines.md).
- Fixes product names and capitalization

The doc updates impact the following Azure doc pages:

- [Container service metrics](https://docs.elastic.co/integrations/azure_metrics/container_service)
- [Database Account metrics](https://docs.elastic.co/integrations/azure_metrics/database_account)
- [Storage Account metrics](https://docs.elastic.co/integrations/azure_metrics/storage_account)
- [Virtual machines metrics](https://docs.elastic.co/integrations/azure_metrics/compute_vm)
- [Virtual machines scaleset metrics](https://docs.elastic.co/integrations/azure_metrics/compute_vm_scaleset)

## Related issues

- https://github.com/elastic/integration-docs/issues/332
- https://github.com/elastic/integration-docs/issues/324

### CODE REVIEW

File: packages/azure_metrics/_dev/build/docs/database_account.md [hunk 1]
```diff
@@ -1,52 +1,18 @@
 # Azure Database Account Integration
 
+The Azure Database Account data stream collects and aggregates database account related metrics from Azure Database Account type resources where it can be used for analysis, visualization, and alerting.
+The Azure Database Account will periodically retrieve the Azure Monitor metrics using the Azure REST APIs as MetricList.
+Additional azure API calls will be executed to retrieve information regarding the resources targeted by the user.
```
Comment (zmoog) [inline packages/azure_metrics/_dev/build/docs/database_account.md:5]:
```suggestion
Additional Azure API calls will be executed to retrieve information regarding the resources targeted by the user.
```

File: packages/azure_metrics/docs/database_account.md [hunk 2]
```diff
@@ -1,52 +1,18 @@
 # Azure Database Account Integration
 
+The Azure Database Account data stream collects and aggregates database account related metrics from Azure Database Account type resources where it can be used for analysis, visualization, and alerting.
+The Azure Database Account will periodically retrieve the Azure Monitor metrics using the Azure REST APIs as MetricList.
+Additional azure API calls will be executed to retrieve information regarding the resources targeted by the user.
```
Comment (zmoog) [inline packages/azure_metrics/docs/database_account.md:5]:
```suggestion
Additional Azure API calls will be executed to retrieve information regarding the resources targeted by the user.
```

**Review by zmoog — APPROVED** [review]
LGTM, I only added a couple of capitalization suggestions.

**Comment by zmoog** [comment]
> LGTM, I only added a couple of capitalization suggestions.

This time I update both docs sources and renders, so it should not break the build 🤞

**Review by muthu-mps — APPROVED** [review]
LGTM!

**Pull request merged**